src/melee/ft/chara/ftCLink/ftclink.h

```diff
@@ -4,6 +4,7 @@
 #include <melee/ft/fighter.h>
 #include <sysdolphin/baselib/gobj.h>
 
+/// @todo this is #ftLinkAttributes.
```

Comment:
do you mean ftCLinkAttributes? Or are they the same for both

Reply:
Yeah, they are the same struct. Same applies to all clone characters it seems like (and Nana).

Reply:
Interesting, thanks